src/terrain/routes/instantlaunches.clj

```diff
@@ -22,26 +22,38 @@
        (context "/defaults" []
 
          (GET "/latest" []
+           :middleware  [require-authentication]
            :summary LatestILMappingsDefaultsSummary
            :description LatestILMappingsDefaultsDescription
            :return DefaultInstantLaunchMapping
            (ok (latest-instant-launch-mappings-defaults)))))
 
      (GET "/" []
+       :middleware  [require-authentication]
        :summary ListInstantLaunchesSummary
        :description ListInstantLaunchesDescription
        :return InstantLaunchList
        (ok (get-instant-launch-list)))
 
      (GET "/full" []
+       :middleware  [require-authentication]
        :summary ListFullInstantLaunchesSummary
        :description ListFullInstantLaunchesDescription
        :return FullInstantLaunchList
        (ok (get-full-instant-launch-list)))
 
+     (context "/metadata" []
+       (GET "/full" []
+         :summary ListFullMetadataSummary
+         :description ListFullMetadataDescription
+         :query [query MetadataListingQueryMap]
+         :return FullInstantLaunchList
+         (ok (list-full-metadata query (or (:username current-user) "anonymous")))))
```

Comment:
I don't think there needs to be another `context` here, since it's only used by 1 endpoint:
```suggestion
       (GET "/metadata/full" []
         :summary ListFullMetadataSummary
         :description ListFullMetadataDescription
         :query [query MetadataListingQueryMap]
         :return FullInstantLaunchList
         (ok (list-full-metadata query (or (:username current-user) "anonymous"))))
```